core/conn/jdbc_type2/native/SrvrSmd.cpp

```diff
@@ -300,6 +301,15 @@ void odbc_SQLSvc_GetSQLCatalogs_sme_(
 		inputParam[2] = schemaNmNoEsc;
 		inputParam[3] = expSchemaNm;
 	    inputParam[4] = NULL;
+	    if(uniqueness == 0)
```

Comment:
Can you please add some comment that this is related to SQL_INDEX_UNIQUE(0) or use the same ?